app/ssttpeligibility/SelfServiceTimeToPayController.scala

```diff
@@ -107,6 +109,19 @@ class SelfServiceTimeToPayController @Inject() (
     }
   }
 
+  def confidenceUplift: Action[AnyContent] = as.action { implicit request =>
```

Comment:
Do we really need this extra endpoint in the microservice? Maybe it's easier to build the redirect in the action?